erroraway.properties

```diff
@@ -0,0 +1,12 @@
+category = External Analysers
+description = Analyze Java Code with ErrorProne, NullAway, errorprone-slf4j and Picnic ErrorProne support
+publicVersions = 2.2.2
```

Comment:
please add a `homepageUrl`

Reply:
I've added the homepage and issuer tracker urls, thanks!